README.md

```diff
@@ -1,165 +1,92 @@
-| Stress testing         | Run millions of inferences across thousands of models and log all the results to find the bugs in a HW/SW stack. | `turnkey timm/*.py --iterations 1000 --device MY_DEVICE --runtime MY_RUNTIME` |
-| Model insights         | Analyze a model to learn its parameter count, input shapes, which ONNX ops it uses, etc. | `turnkey model.py` <br /> `turnkey cache stats MY_BUILD`|
+Breaking it down:
 
+1. `turnkey -i bert.py` feeds [`bert.py`](https://github.com/onnx/turnkeyml/blob/main/models/transformers/bert.py), a minimal PyTorch script that instantiates BERT, into the tool sequence, starting with...
+1. `discover` is a tool that finds the PyTorch model in a script and passes it to the next tool, which is...
+1. `export-pytorch`, which takes a PyTorch model and converts it to an ONNX model, then passes it to...
+1. `optimize-ort`, which uses `onnxruntime` to optimize the model's compute graph, then passes it to...
+1. `convert-fp16`, which uses `onnxmltools` to convert the ONNX file into fp16.
+1. Finally, the result is printed, and we can see that the requested `.onnx` files have been produced.
 
+All without writing a single line of code or learning how to use any of the underlying ONNX ecosystem tools 🚀
 
-## Demo
+## How It Works
 
-Let's say you have a Python script that includes a PyTorch model. Maybe you downloaded the model from Huggingface, grabbed it from our corpus, or wrote it yourself. Doesn't matter, just call `turnkey` and get to work.   
+The `turnkey` CLI provides a set of `Tools` that users can invoke in a `Sequence`. The first `Tool` takes the input (`-i`), performs some action, and passes its state to the next `Tool` in the `Sequence`.
 
-The `turnkey` CLI will analyze your script, find the model(s), run an ONNX toolchain on the model, and execute the resulting ONNX file in CPU hardware:
+You can read the `Sequence` out like a sentence. For example, the demo command above was:
 
 ```
-> turnkey bert.py
+> turnkey -i bert.py discover export-pytorch optimize-ort convert-fp16
 ```
 
-```
-Models discovered during profiling:
-
-bert.py:
-        model (executed 1x)
-                Model Type:     Pytorch (torch.nn.Module)
-                Class:          BertModel (<class 'transformers.models.bert.modeling_bert.BertModel'>)
-                Location:       /home/jfowers/turnkeyml/models/transformers/bert.py, line 23
-                Parameters:     109,482,240 (417.64 MB)
-                Input Shape:    'attention_mask': (1, 128), 'input_ids': (1, 128)
-                Hash:           bf722986
-                Build dir:      /home/jfowers/.cache/turnkey/bert_bf722986
-                Status:         Successfully benchmarked on AMD Ryzen 9 7940HS w/ Radeon 780M Graphics (ort v1.15.1) 
-                                Mean Latency:   44.168  milliseconds (ms)
-                                Throughput:     22.6    inferences per second (IPS)
-```
```

Comment:
This will get much easier to maintain without those.

Reply:
Yeah that was not fun!